### ISSUE TITLE
Favicon

### ISSUE THREAD

**Issue by sbddesign** [ATLBitLab/twelvecash#39]
Custom favicon

**Comment by paulosacramento** [comment]
Here is my suggestion for the favicon. After testing a few other alternatives, this is my favourite.
![android-chrome-192x192](https://github.com/user-attachments/assets/2d0bf925-f3cc-40c7-96e3-a427df725cba)

Favicon package generated with https://realfavicongenerator.net
[Uploading favicon_package.zip…]()

The tool said that these should be added to the **<head>** section of the page:
```
<link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
<link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
<link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
<link rel="manifest" href="/site.webmanifest">
<link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5">
<meta name="msapplication-TileColor" content="#da532c">
<meta name="theme-color" content="#ffffff">
```

**Comment by paulosacramento** [comment]
Package here:
[favicon_package.zip](https://github.com/user-attachments/files/16738166/favicon_package.zip)

**Comment by sbddesign** [comment]
@paulosacramento do you think we could try one with a little bit of corner rounding?

**Comment by sbddesign** [comment]
@paulosacramento I can handle generating any ICOs and adding the code (it's a little different becaue it needs to work with NextJS's metadata scheme). But if you can just round the corners and give me a PNG of it, I can add it into the code. Thanks for taking the initiative!

**Comment by paulosacramento** [comment]
@sbddesign Do you think these will work?
[favicon.zip](https://github.com/user-attachments/files/16825011/favicon.zip)

**Comment by sbddesign** [comment]
Great, thanks, I will add it

**Comment by sbddesign** [comment]
This will get merged in #44
